.gitignore

```diff
@@ -2,3 +2,5 @@
 composer.lock
 
 .phpunit.cache
+
+/.idea
```

Comment:
please revert this, you should can ignore ide based config globally

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file

Reply:
thanks for the link, was not aware of something like this :see_no_evil: